.changelog/24785.txt

```diff
@@ -0,0 +1,11 @@
+```release-note:breaking-change
+api: QuotaSpec.RegionLimit is now of type QuotaResources instead of Resources
+```
+
+```release-note:deprecation
+api: QuotaSpec.VariablesLimit field is deprecated in lieu of QuotaSpec.RegionLimit.Storage.Variables and will be removed in Nomad 1.12.0
```

Comment:
I think you're using "in lieu" as opposite its meaning.  If I eat peanuts in lieu of cashews, it means I don't have cashews (which I prefer), so I must eat peanuts instead.  Rule of thumb: "in lieu of" = "instead of".

```suggestion
api: QuotaSpec.VariablesLimit field is deprecated and will be removed in Nomad 1.12.0. Use QuotaSpec.RegionLimit.Storage.Variables instead.
```

Reply:
:facepalm: